src/views/css/main.css

```diff
@@ -160,3 +160,11 @@ body {
 .tab-content {
     margin-top: 10px;
 }
+
+.emoji-class {
+    width: 20%;
+}
+
+.customer-class {
+    width: 80%;
+}
```

Comment:
You probably don't need those css classes. It can be done with Bootstrap out of the box.
```html
<li class="list-group-item">
  <div class="row">
    <div class="col-xs-3">
      <!-- emoji -->
    </div>
    <div class="col-xs-9">
      <!-- other stuff -->
    </div>
  </div>
</li>
```

Reply:
Ahh yeah, been a while since I did this. Thanks